plugin-guide.md

```diff
@@ -116,9 +116,7 @@ shared directory instead of, say, `/tmp` and ensure the right permissions are
 set up.
 These are not the responsibility of the plugin author.
 
-The class `BaseJsonInputProvisioner` is a partial implementation that can store
-crash recovery information in a JSON object of the implementor's choosing,
-making recovery easier.
+This plugin type uses the operations API.
```

Comment:
Can these be a link to the section? if markdown can do that?

Reply:
Done.